lib/universal-search.js

```diff
@@ -88,22 +88,47 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
 
+    //Const for main stylesheet.
```

Comment:
Adding informative comments is tricky. You don't want to just restate what the code says, you want to provide slightly higher-level context. So, a more useful comment here might be:

`// Insert the main stylesheet with platform-independent styles.`